PEN-769: Fixed more bugs in section title

beltrancaliz opened the pull request:
Fixed remaining issues in Section Title
[Jira ticket](https://arcpublishing.atlassian.net/browse/PEN-769)

beltrancaliz committed:
Fixed more bugs in section title

beltrancaliz committed:
Merge branch 'master' into PEN-769-Section-Block-Update

beltrancaliz committed:
Added back conditional check

JackHowa reviewed (approved):
Looks good. Makes sense that you added another check

beltrancaliz committed:
Publish

 - @wpmedia/article-body-block@2.0.11
 - @wpmedia/article-tag-block@2.0.11
 - @wpmedia/author-bio-block@1.0.11
 - @wpmedia/author-content-source-block@1.0.11
 - @wpmedia/byline-block@1.0.11
 - @wpmedia/card-list-block@1.0.11
 - @wpmedia/collections-content-source-block@1.1.3
 - @wpmedia/content-api-source-block@1.0.11
 - @wpmedia/date-block@2.0.11
 - @wpmedia/default-output-block@2.2.2
 - @wpmedia/double-chain-block@1.0.11
 - @wpmedia/extra-large-manual-promo-block@1.0.11
 - @wpmedia/extra-large-promo-block@1.0.12
 - @wpmedia/footer-block@2.0.11
 - @wpmedia/full-author-bio-block@1.0.11
 - @wpmedia/gallery-block@1.0.11
 - @wpmedia/header-nav-block@2.0.11
 - @wpmedia/headline-block@2.0.11
 - @wpmedia/htmlbox-block@1.0.11
 - @wpmedia/large-manual-promo-block@1.0.11
 - @wpmedia/large-promo-block@1.0.12
 - @wpmedia/lead-art-block@2.0.11
 - @wpmedia/links-bar-block@2.0.11
 - @wpmedia/masthead-block@1.0.11
 - @wpmedia/medium-manual-promo-block@1.0.11
 - @wpmedia/medium-promo-block@1.0.12
 - @wpmedia/numbered-list-block@3.0.11
 - @wpmedia/overline-block@1.0.11
 - @wpmedia/quad-chain-block@1.0.6
 - @wpmedia/results-list-block@2.0.11
 - @wpmedia/right-rail-block@1.1.2
 - @wpmedia/search-content-source-block@1.0.11
 - @wpmedia/search-results-list-block@0.2.6
 - @wpmedia/section-title-block@1.1.6
 - @wpmedia/share-bar-block@2.0.11
 - @wpmedia/shared-styles@1.0.11
 - @wpmedia/simple-list-block@2.0.11
 - @wpmedia/single-chain-block@1.0.11
 - @wpmedia/site-hierarchy-content-block@3.1.10
 - @wpmedia/small-manual-promo-block@1.0.11
 - @wpmedia/small-promo-block@1.0.12
 - @wpmedia/story-feed-author-content-source-block@1.0.11
 - @wpmedia/story-feed-query-content-source-block@1.0.11
 - @wpmedia/story-feed-sections-content-source-block@1.0.11
 - @wpmedia/story-feed-tag-content-source-block@1.0.11
 - @wpmedia/subheadline-block@2.0.11
 - @wpmedia/tag-title-block@1.0.11
 - @wpmedia/tags-content-source-block@1.0.11
 - @wpmedia/top-table-list-block@1.0.11
 - @wpmedia/triple-chain-block@1.0.11
 - @wpmedia/unpublished-content-source-block@1.0.11
 - @wpmedia/video-player-block@1.0.11
